spec/capitalize_decorator_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../capitalize_decorator'
+
+describe CapitalizeDecorator do
+  context 'Unit Test For Classroom Class' do
+    before(:context) do
+      @capitalize = CapitalizeDecorator.new('Mark')
+    end
+    it 'Creates a new instance' do
+      expect(@capitalize).to be_an_instance_of(CapitalizeDecorator)
+    end
+  end
```

Comment:
- [ ] Good job with test coverage, but since we were aiming for 100% coverage, this is to make sure our app does not break in production, and it a [requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md)for this project,  "`Add unit tests for all the methods in all the classes`", please it would be nice to cover the following methods is the `CapitalizeDecorator ` with tests: 

- `correct_name`: This method should return a name with the first letter capitalized.